internal/service/verify/data_source_voice_phrase_content.go

```diff
@@ -0,0 +1,214 @@
+	VoicePhraseContentId types.String `tfsdk:"voice_phrase_content_id"`
+	VoicePhraseId        types.String `tfsdk:"voice_phrase_id"`
+	Locale               types.String `tfsdk:"locale"`
+	Content              types.String `tfsdk:"content"`
+	CreatedAt            types.String `tfsdk:"created_at"`
+	UpdatedAt            types.String `tfsdk:"updated_at"`
+}
+
+// Framework interfaces
+var (
+	_ datasource.DataSource = &VoicePhraseContentDataSource{}
+)
+
+// New Object
+func NewVoicePhraseContentDataSource() datasource.DataSource {
+	return &VoicePhraseContentDataSource{}
+}
+
+// Metadata
+func (r *VoicePhraseContentDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_voice_phrase_content"
+}
+
+func (r *VoicePhraseContentDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+
+	// schema descriptions and validation settings
+
+	// P1 Platform does not set a traditional UUID as the default phrase ID value
+	const defaultVoicePhraseId = "exceptional_experiences"
+
+	phraseIdDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"For a customer-defined phrase, the identifier (UUID) of the `voice_phrase` associated with the `voice_phrase_content` configuration. For pre-defined phrases, a string value.",
+	)
+
+	contentDescription := framework.SchemaAttributeDescriptionFromMarkdown(
+		"The phrase a user must speak as part of the voice enrollment or verification. The phrase must be written in the language and character set required by the language specified in the `locale` property.",
+	)
+
+	resp.Schema = schema.Schema{
+		// This description is used by the documentation generator and the language server.
+		Description: "Data source to find PingOne Voice Phrase Contents by its Voice Phrase Content Id for a speicific PingOne Voice Phrase Id.",
+
+		Attributes: map[string]schema.Attribute{
+			"id": framework.Attr_ID(),
+
+			"environment_id": framework.Attr_LinkID(
+				framework.SchemaAttributeDescriptionFromMarkdown("PingOne environment identifier (UUID) in which the verify voice phrase exists."),
+			),
+
+			"voice_phrase_content_id": schema.StringAttribute{
+				Description: "Identifier (UUID) associated with the voice phrase content.",
+				Required:    true,
+				Validators: []validator.String{
+					validation.P1ResourceIDValidator(),
+				},
+			},
+
+			"voice_phrase_id": schema.StringAttribute{
+				Description:         phraseIdDescription.Description,
+				MarkdownDescription: phraseIdDescription.MarkdownDescription,
+				Required:            true,
+				Validators: []validator.String{
+					stringvalidator.Any(
+						validation.P1ResourceIDValidator(),
+						stringvalidator.RegexMatches(regexp.MustCompile(defaultVoicePhraseId), "Unexpected error with the pre-defined, default value. Please report this issue to the provider maintainers."),
```

Comment:
The error message on the regex is inappropriate, and my logic on error handling insufficient.  The challenge is there are two default `voice_phrases` in verify, and instead of a UUID, they have an `id` of either `pingone_davinci_nocode` or `exceptional_experiences`.  

A customer may want to use one of the default objects for their `voice_phrase_content`, so I was trying to allow them to do so. 

I will update the logic to handle the defaults more clearly.